### ISSUE TITLE
ERROR: LoadError: UndefVarError: `stile` not defined in `PPTX`

### ISSUE THREAD

**Issue by ysaereve** [ASML-Labs/PPTX.jl#59]
Thank you so much for creating this wonderful package! 

Got the following error during test drive PPTX.jl
ERROR: LoadError: UndefVarError: `stile` not defined in `PPTX`

TDPPTX.jl
``` 
text = TextBox(content="Look its a vegetable!", offset_x=100, offset_y=140, style = Dict("bold" => true, "italic" => true))
```

PPTX.jl/src/TextBox.jl at main · ASML-Labs/PPTX.jl
https://github.com/ASML-Labs/PPTX.jl/blob/main/src/TextBox.jl
```
126     push!(stile, Dict("b" => "1"))
```

```
ERROR: LoadError: UndefVarError: `stile` not defined in `PPTX`
Stacktrace:
  [1] text_style_xml(t::PPTX.TextBody)
    @ PPTX ~/.julia/packages/PPTX/IjAGP/src/TextBox.jl:126
  [2] make_textbody_xml(t::PPTX.TextBody, txBodyNameSpace::String)
    @ PPTX ~/.julia/packages/PPTX/IjAGP/src/TextBox.jl:165
  [3] make_textbody_xml
    @ ~/.julia/packages/PPTX/IjAGP/src/TextBox.jl:165 [inlined]
  [4] make_textbody_xml
    @ ~/.julia/packages/PPTX/IjAGP/src/TextBox.jl:181 [inlined]
  [5] make_xml(t::TextBox, id::Int64)
    @ PPTX ~/.julia/packages/PPTX/IjAGP/src/TextBox.jl:159
  [6] make_slide(s::Slide)
    @ PPTX ~/.julia/packages/PPTX/IjAGP/src/Slide.jl:100
  [7] write_slides!(w::ZipArchives.ZipWriter{IOStream}, p::Presentation, template::ZipArchives.ZipReader{Vector{UInt8}})
    @ PPTX ~/.julia/packages/PPTX/IjAGP/src/write.jl:24
  [8] (::PPTX.var"#22#24"{Presentation, ZipArchives.ZipReader{Vector{UInt8}}})(w::ZipArchives.ZipWriter{IOStream})
    @ PPTX ~/.julia/packages/PPTX/IjAGP/src/write.jl:178
  [9] ZipArchives.ZipWriter(f::PPTX.var"#22#24"{Presentation, ZipArchives.ZipReader{Vector{UInt8}}}, io::IOStream; zip_kwargs::@Kwargs{own_io::Bool})
    @ ZipArchives ~/.julia/packages/ZipArchives/yhcc0/src/writer.jl:55
 [10] ZipWriter
    @ ~/.julia/packages/ZipArchives/yhcc0/src/writer.jl:52 [inlined]
 [11] (::PPTX.var"#21#23"{Bool, Presentation, ZipArchives.ZipReader{Vector{UInt8}}})(temp_path::String, temp_out::IOStream)
    @ PPTX ~/.julia/packages/PPTX/IjAGP/src/write.jl:173
 [12] mktemp(fn::PPTX.var"#21#23"{Bool, Presentation, ZipArchives.ZipReader{Vector{UInt8}}}, parent::String)
    @ Base.Filesystem ./file.jl:790
 [13] write(filepath::String, p::Presentation; overwrite::Bool, open_ppt::Bool, template_path::String)
    @ PPTX ~/.julia/packages/PPTX/IjAGP/src/write.jl:172
 [14] top-level scope
    @ /Projects/Babel/Julia/TDPPTX/TDPPTX.jl:46
 [15] include(fname::String)
    @ Main ./sysimg.jl:38
 [16] run(debug_session::VSCodeDebugger.DebugAdapter.DebugSession, error_handler::VSCodeDebugger.var"#3#4"{String})
    @ VSCodeDebugger.DebugAdapter ~/.vscode-insiders/extensions/julialang.language-julia-1.124.2/scripts/packages/DebugAdapter/src/packagedef.jl:122
 [17] startdebugger()
    @ VSCodeDebugger ~/.vscode-insiders/extensions/julialang.language-julia-1.124.2/scripts/packages/VSCodeDebugger/src/VSCodeDebugger.jl:45
 [18] top-level scope
    @ ~/.vscode-insiders/extensions/julialang.language-julia-1.124.2/scripts/debugger/run_debugger.jl:12
 [19] include(mod::Module, _path::String)
    @ Base ./Base.jl:557
 [20] exec_options(opts::Base.JLOptions)
    @ Base ./client.jl:323
 [21] _start()
    @ Base ./client.jl:531
```

**Comment by matthijscox-asml** [comment]
Thanks for reporting this bug! Seems you found some untested code.

I'll try to fix it as soon as possible.

**Comment by matthijscox-asml** [comment]
This should be fixed in v0.8.1, should be available to you once the registry PR https://github.com/JuliaRegistries/General/pull/117223 is merged.

**Comment by ysaereve** [comment]
Thank you for your help. I just downloaded v0.9.0. It fixed the reported error.
